Composer/packages/client/src/store/action/storage.ts

```diff
@@ -95,5 +95,15 @@ export const fetchFolderItemsByPath: ActionCreator = async ({ dispatch }, id, pa
 };
 
 export const updateCurrentPath: ActionCreator = async ({ dispatch }, path) => {
-  await httpClient.put(`/storages/currentPath`, { path: path });
+  try {
+    const response = await httpClient.put(`/storages/currentPath`, { path: path });
+    dispatch({
+      type: ActionTypes.GET_STORAGE_SUCCESS,
+      payload: {
+        response,
+      },
+    });
+  } catch (err) {
+    dispatch({ type: ActionTypes.GET_STORAGE_FAILURE, payload: null, error: err });
```

Comment:
You can make it simpler by writing it like this.

Also, is `payload: null` required? If we leave it out, it will be more like `payload: undefined`.

```js
  } catch (error) {
    dispatch({ type: ActionTypes.GET_STORAGE_FAILURE, payload: null, error });
```